### PR TITLE
added coin cursor for scratch

### PR DESCRIPTION
**Pull request by odax**
# Description

Formatted and added a .png coin for the mouse cursor when hovering over .ScratchCard__Container inside CountryBorder.js

## Type of change

- [x] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

Passes local tests

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] My code has been reviewed by at least one peer
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: client/src/Components/CountryBorder/CountryBorder.less [hunk 1]
```diff
@@ -2,6 +2,10 @@
   border: 1px solid rgba(0, 0, 0, 0.2);
   box-sizing: border-box;
   background: rgba(100, 99, 99, 0.2);
+  .ScratchCard__Container {
+    border: 3px solid red;
```
Comment (nikhilkamineni) [inline client/src/Components/CountryBorder/CountryBorder.less]: Looks good. You could probably remove this border though.

File: client/src/Components/CountryBorder/CountryBorder.less [hunk 2]
```diff
@@ -2,6 +2,10 @@
...
+    cursor: url(coincursor.png), auto;
```
Comment (nikhilkamineni) [inline client/src/Components/CountryBorder/CountryBorder.less]: Coin looks good, but it looks a little too big to me. Maybe you could make it a little smaller?

Comment (odax) [reply client/src/Components/CountryBorder/CountryBorder.less]: Reduced size by 25%

File: client/src/Components/CountryBorder/CountryBorder.less (hunk 1 above)
Comment (odax) [reply client/src/Components/CountryBorder/CountryBorder.less]: Removed

**Comment by nikhilkamineni** [comment]
Cool. Probably need to adjust the offset a little, but lets merge this and do that when we get to the styling.

**Pull request merged**